### PR TITLE
WL-3940: Move Find It and other links to the rhs

### PR DESCRIPTION
**Pull request by nicholaswilson100**
I'm including citations.css  in the page so we can have a css just for the Reading List tool

### CODE REVIEW

**Comment by nicholaswilson100** [comment]
Linked to https://github.com/ox-it/wl-reference/pull/89

**Comment by nicholaswilson100** [comment]
It looks like it might be quite useful to put quite a few Jiras on this pull request which muddles things I know but a lot of them reuse and build on code from previous jiras

File: citations-util/util/src/bundle/citations_en_GB.properties [hunk 1]
```diff
@@ -1,4 +1,6 @@
-list.title			= Reading List
+list.title			=
+list.title.bkgd.col = #002147
```
Comment (buckett) [inline citations-util/util/src/bundle/citations_en_GB.properties]: Config shouldn't go in i18n files.

File: citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java [hunk 2]
```diff
@@ -464,7 +466,8 @@ protected void handleViewRequest(HttpServletRequest req, HttpServletResponse res
     			// show detailed info
     			out.println("\t\t<div id=\"details_" + escapedId + "\" class=\"citationDetails\" style=\"display: none;\">");
        			out.println("\t\t\t<table class=\"listHier lines nolines\" style=\"margin-left: 2em;\" cellpadding=\"0\" cellspacing=\"0\">");
-	     			
+
+    			out.println("\t\t\t\t<tr>\n\t\t\t\t\t<td class=\"attach header\" style=\"color:" + rb.getFormattedMessage("list.title.bkgd.col")  + "\" colspan=\"2\"><strong>" + "Item details" + "</strong></td></tr>");
```
Comment (buckett) [inline citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java]: `Item Details:` should come from i18n bundle.

File: citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java [hunk 3]
```diff
@@ -360,10 +362,10 @@ protected void handleViewRequest(HttpServletRequest req, HttpServletResponse res
     			String href = citation.hasPreferredUrl() ? citation.getCustomUrl(citation.getPreferredUrlId()) : citation.getOpenurl();
     			
     			out.println("\t\t<td headers=\"details\">");
-    			out.println("\t\t\t<a href=\"" + Validator.escapeHtml(href) + "\" target=\"_blank\">" + Validator.escapeHtml( (String)citation.getCitationProperty( Schema.TITLE, true ) ) + "</a><br />");
-    			out.println("\t\t\t\t" + Validator.escapeHtml( citation.getCreator() ) );
-    			out.println("\t\t\t\t" + Validator.escapeHtml( citation.getSource() ) );
-    			out.println("\t\t\t<div class=\"itemAction\">");
+    			out.println("\t\t\t<div class=\"floatLeft fullWidth\"><div class=\"floatLeft\"><a href=\"" + Validator.escapeHtml(href) + "\" target=\"_blank\">" + Validator.escapeHtml( (String)citation.getCitationProperty( Schema.TITLE, true ) ) + "</a>");
```
Comment (buckett) [inline citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java]: Ideally CSS names should be semantic and actual layout used should be in the CSS files.

File: citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java [hunk 4]
```diff
@@ -314,11 +315,12 @@ protected void handleViewRequest(HttpServletRequest req, HttpServletResponse res
 
     		List<Citation> citations = collection.getCitations();
     		
-    		out.println("<div class=\"portletBody\">\n\t<div class=\"indnt1\">");
-    		out.println("\t<h3>" + rb.getString("list.title") + ": " + Validator.escapeHtml(title) + "</h3>");
+    		out.println("<div class=\"portletBody\">\n\t<div class=\"indnt1 citationList\">");
+    		out.println("\t<div style=\"background-color:" + rb.getFormattedMessage("list.title.bkgd.col") + "; padding: 15px;\"><h3 style=\"color:" +
```
Comment (buckett) [inline citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java]: Put the padding in the CSS file?

Comment (nicholaswilson100) [inline citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java]: ok

File: citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java (hunk 3 above)
Comment (nicholaswilson100) [inline citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java]: yup

File: citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java (hunk 2 above)
Comment (nicholaswilson100) [inline citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java]: ok

File: citations-util/util/src/bundle/citations_en_GB.properties (hunk 1 above)
Comment (nicholaswilson100) [inline citations-util/util/src/bundle/citations_en_GB.properties]: ok - all done!

**Comment by buckett** [comment]
The problem now is that you're last commit isn't WL-3940, it a combination of all the earlier issues.

**Pull request merged**